app/vmauth/auth_config.go

```diff
@@ -309,12 +327,22 @@ func parseAuthConfig(data []byte) (map[string]*UserInfo, error) {
 			ui.requests = metrics.GetOrCreateCounter(fmt.Sprintf(`vmauth_user_requests_total{username=%q}`, name))
 		}
 		if ui.Username != "" {
-			name := ui.Username
+			name = ui.Username
 			if ui.Name != "" {
 				name = ui.Name
 			}
 			ui.requests = metrics.GetOrCreateCounter(fmt.Sprintf(`vmauth_user_requests_total{username=%q}`, name))
 		}
+		if ui.MaxConcurrentRequests > 0 {
+			ui.limitReached = metrics.NewCounter(fmt.Sprintf(`vmauth_concurrent_requests_limit_reached_total{username=%q}`, name))
```

Comment:
Could It panic after config reload?

Reply:
See GetOrCreateCounter

Reply:
Yes, I forgot about `GetOrCreateCounter`, will update soon